src/Services/Identity/Identity.Presentation/ExceptionHandling/ExceptionHandlingMiddleware.cs

```diff
@@ -44,7 +44,7 @@ public async Task InvokeAsync(HttpContext context)
                 Status = StatusCodes.Status400BadRequest,
                 Type = "https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.1",
                 Title = "Existing user",
-                Detail = exception.Message
+                Detail = exception.Message,
             };
 
             context.Response.StatusCode = StatusCodes.Status400BadRequest;
```

Comment:
context.Response.StatusCode and await context.Response.WriteAsJsonAsync(problemDetails); is duplicated

Reply:
Would you mind describing problem more?